src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java

```diff
@@ -408,4 +433,41 @@ public JsonObject search(String query, String index, String type) throws IOExcep
   public void close() {
     client.shutdownClient();
   }
+
+  public enum WriteMethod {
+    INSERT,
+    UPDATE, // ignored for now
```

Comment:
We should remove this `UPDATE` literal, because it is unnecessary for now and should be completely unused. After all, we may never need to add it back in the future.